libs/integration-tests-ocm/package.json

```
@@ -0,0 +1,37 @@
+{
+  "name": "@openshift-assisted/integration-tests-ocm",
+  "version": "1.0.0",
+  "private": true,
+  "license": "Apache-2.0",
+  "dependencies": {
+    "@types/file-saver": "^2.0.1",
+    "@types/jest": "^24.0.0",
+    "@types/lodash": "^4.14.121",
+    "@types/node": "^12.0.0",
+    "@types/uuid": "^8.0.0",
+    "file-saver": "^2.0.2",
+    "http-proxy-middleware": "^1.0.3",
+    "lodash": "^4.17.15",
+    "uuid": "^8.1.0",
+    "yaml": "^1.7.2"
+  },
+  "scripts": {
+    "build:assisted-ui": "yarn workspace @openshift-assisted/assisted-ui run build",
+    "preview:assisted-ui": "yarn workspace @openshift-assisted/assisted-ui run preview",
+    "test:assisted-ui": "yarn run --top-level start-server-and-test preview:assisted-ui http://localhost:4173 test",
```

Make it use yarn instead of npm
```suggestion
    "test:assisted-ui": "yarn run -T start-server-and-test 'yarn preview:assisted-ui' http://localhost:4173 'yarn test'",
```